ingest/src/main/java/com/microsoft/azure/kusto/ingest/IngestClientImpl.java

```diff
@@ -214,54 +210,26 @@ private String genBlobName(String fileName, String databaseName, String tableNam
         return String.format("%s__%s__%s__%s", databaseName, tableName, UUID.randomUUID().toString(), fileName);
     }
 
-    public IngestionResult ingestFromResultSet(
-            ResultSetSourceInfo resultSetSourceInfo, IngestionProperties ingestionProperties)
+    public IngestionResult ingestFromResultSet(ResultSetSourceInfo resultSetSourceInfo, IngestionProperties ingestionProperties)
```

Comment:
I would cut off the line at throws, or reformat a bit
```suggestion
    public IngestionResult ingestFromResultSet(
               ResultSetSourceInfo resultSetSourceInfo, 
               IngestionProperties ingestionProperties
     ) throws ...
```